### ISSUE TITLE
Incorrect test for reduce without an accumulator

### ISSUE THREAD

**Issue by maxcell** [learn-co-curriculum/js-functional-library-project#15]
Just marking this so I can easily come back to this.

The accumulator test as is says it should return 30, however, if done in the console using the built in `reduce`, it says it should be 28.

**Comment by sgharms** [comment]
@maxcell Is this still a valid issue? I couldn't find a reduce() call that seems to line up to this?

**Comment by maxcell** [comment]
@sgharms Yup still there! [The line in question is this](https://github.com/learn-co-curriculum/js-functional-library-project/blob/master/test/indexTest.js#L69)

Given that the array and callback is written as:
```
const testArr = unmodifiedTestArr.slice() // arr is [1, 2, 3, 4]
const callback = (acc, val, collection) => (acc + (val * 3))
```

This would mean that the stages should go:

  1. Accumulator is 1, current value is 2; Result should be: 7
  2. Accumulator is 7, current value is 3; Result should be: 16
  3. Accumulator is 6, current value is 4; Result should be 28

The ending result should be 28.

Also reproducible by:
```
const callback = (acc, val, collection) => (acc + (val * 3))
[1,2,3,4].reduce(callback) // 28
```

**Comment by sgharms** [comment]
@maxcell Fixed as of: c66f4e8. Thanks!
